src/http/server/handlers/websocket_handler.cr

```diff
@@ -15,20 +15,28 @@ class HTTP::WebSocketHandler
 
   def call(context)
     if websocket_upgrade_request? context.request
-      key = context.request.headers["Sec-Websocket-Key"]
+      response = context.response
+
+      version = context.request.headers["Sec-WebSocket-Version"]?
+      unless version == WebSocket::Protocol::VERSION.to_s
```

Comment:
Let's have `WebSocket::Protocol::VERSION_STRING` too, or something like that. I'd like to avoid a `to_s` call for each socket upgrade.

Reply:
Good catch :+1: It's only used as a string anyway, so I'll just change the datatype of the constant.